src/models/disassemblyoutput.cpp

```diff
@@ -56,14 +60,15 @@ static QVector<DisassemblyOutput::DisassemblyLine> objdumpParse(const QByteArray
             continue;
         }
 
-        const auto asmTokens = asmLine.splitRef(QLatin1Char(':'));
-        const auto addrLine = asmTokens.value(0).trimmed();
-
-        bool ok = false;
-        const auto addr = addrLine.toULongLong(&ok, 16);
-        if (!ok) {
-            qWarning() << "unhandled asm line format:" << addrLine;
-            continue;
+        quint64 addr = 0;
+        const auto match = disassemblyRegex.match(asmLine);
+        if (match.hasMatch()) {
+            bool ok = false;
+            addr = match.capturedRef(1).toULongLong(&ok, 16);
+            if (!ok) {
+                qWarning() << "unhandled asm line format:" << asmLine;
+                continue;
+            }
         }
```

Comment:
and if there's no match, what then? is it fine to add the line with `addr=0` - is that already handled elsewhere correctly, i.e. we don't want to show `0` for all these lines in the view?

Reply:
I added a check to the disassembly model to skip the cost calculation if addr is 0. This should be fine since normally a process can't access that area.